### PR TITLE
feat: Audit logs PLUTO-952

### PR DESCRIPTION
**Pull request by claudiacarpinteiro**
Adds content for the new audit logs capability to the product documentation.

### :eyes: Live preview

- https://pluto-952-audit-logs--docs-codacy.netlify.app/organizations/audit-logs-for-organizations/
- https://pluto-952-audit-logs--docs-codacy.netlify.app/organizations/roles-and-permissions-for-organizations/
- https://pluto-952-audit-logs--docs-codacy.netlify.app/faq/general/does-codacy-keep-audit-logs/

### :construction: To do
-   [x] Create new content
-   [x] Solve TODOs
-   [x] Request validation
-   [ ] Apply feedback
-   [ ] Wait for feature release
-   [ ] Remove **don't merge** label before merging

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -635,6 +635,7 @@ nav:
                 - faq/general/how-does-codacy-support-bitbucket-server.md
                 - faq/general/how-does-codacy-keep-my-data-secure.md
                 - faq/general/how-does-codacy-protect-my-privacy.md
+                - faq/general/which-events-does-codacy-log.md
```
Comment (nfcodacy) [inline mkdocs.yml:638]: **question:** Should this have a title that gives more the idea that this feature is not accessible by default to everyone, more like "which events codacy can log"?

Comment (claudiacarpinteiro) [reply mkdocs.yml]: Thanks, @nfcodacy! Updated. 👍

File: docs/codacy-api/examples/obtaining-audit-log-events-for-organization.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+description: Example of how to obtain audit log events for an organization using the Codacy API endpoint listAuditLogsForOrganization.
+---
+
+# Obtaining audit log events for an organization
```
Comment (claudiacarpinteiro) [inline docs/codacy-api/examples/obtaining-audit-log-events-for-organization.md:5]: @franciscoovazevedo @geirolz Let me know if this example makes sense. 
We are already providing some [similar examples](https://docs.codacy.com/codacy-api/examples/obtaining-code-quality-metrics-for-files/) for other endpoints.

File: docs/organizations/audit-logs-for-organizations.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+|Delete account API token|Account API token deleted|`user.tokens.delete`|
+|?Add Git provider (depends on effort)?|New Git provider added to Codacy||
+
+### Organization
+
+|Event|Description|Action|
+|-----|-----------|------|
+|[Add organization](./what-are-organizations.md#adding-an-organization)|Organization added to Codacy|`organizations.create`|
+|[Add people](./managing-people.md#adding-people) to organization|New people added to the organization|`organizations.people.create`|
+|?Remove person from organization (depends on effort)?|Person removed from the organization|`organizations.people.delete`|
```
Comment (franciscoovazevedo) [inline docs/organizations/audit-logs-for-organizations.md:43]: we are not auditing this one.

File: docs/organizations/audit-logs-for-organizations.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+|[Join organization](./managing-people.md#joining)|User joined the organization|`organizations.join`|
+|Assign [organization manager](./roles-and-permissions-for-organizations.md#managing-the-organization-manager-role)|Organization manager role assigned to a team member|`organizations/security/managers.create`|
+|Revoke organization manager|Organization manager role revoked from a team member|`organizations/security/managers.delete`|
```
Comment (franciscoovazevedo) [inline docs/organizations/audit-logs-for-organizations.md:46]: `organizations.security.managers.delete`

File: docs/organizations/audit-logs-for-organizations.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+|?Update repository management permissions?|Repository management permissions updated||
```
[the remark this answers is not in the document]
Comment (claudiacarpinteiro) [reply docs/organizations/audit-logs-for-organizations.md]: Adding this one back, as we are auditing it.

File: docs/codacy-api/examples/obtaining-audit-log-events-for-organization.md (hunk 2 above)
Comment (claudiacarpinteiro) [reply docs/codacy-api/examples/obtaining-audit-log-events-for-organization.md]: I removed the example, as it doesn't add much value.

**Review by franciscoovazevedo — APPROVED** [review]
LGTM

**Review by geirolz — APPROVED** [review]

**Pull request merged**